Cant Compile

build_static.sh fails with:
```
...
make[2]: Leaving directory '/tmp/libvirt-3.8.0/examples'
make[1]: Leaving directory '/tmp/libvirt-3.8.0/examples'
+ sed -i 's/^Libs:.*/& -lnl -ltirpc -lxml2/' /usr/local/lib/pkgconfig/libvirt.pc
+ cd /libvirt-exporter
+ export 'GOPATH=/gopath'
+ go get -d ./...
+ go build --ldflags '-extldflags "-static"' -o libvirt-exporter
# github.com/prometheus/client_golang/prometheus
/gopath/src/github.com/prometheus/client_golang/prometheus/go_collector.go:388:15: undefined: debug.ReadBuildInfo```

--- a/build_static.sh
+++ b/build_static.sh
@@ -30,7 +30,12 @@ sed -i 's/^Libs:.*/& -lnl -ltirpc -lxml2/' /usr/local/lib/pkgconfig/libvirt.pc
 # Build the libvirt-exporter.
 cd /libvirt-exporter
 export GOPATH=/gopath
-go get -d ./...
+mkdir -p /gopath/
+ln -s /libvirt-exporter/vendor /gopath/src
+mkdir -p /gopath/src/github.com/AlexZzz
+ln -s /libvirt-exporter /gopath/src/github.com/AlexZzz/libvirt-exporter
 go build --ldflags '-extldflags "-static"' -o libvirt-exporter
+unlink /gopath/src/github.com/AlexZzz/libvirt-exporter
+rmdir /gopath/src/github.com/AlexZzz
 strip libvirt-exporter
 EOF

